PowerPointLabs/PowerPointLabs/ShapesLab/CustomShapePane.cs

```diff
@@ -1677,6 +1712,41 @@ private void AddShapeButton_Click(object sender, EventArgs e)
 
             AddShapeFromSelection(selection, addIn);
         }
+
+        // A disabled button cannot respond to any events.
+        // Thus we register the event to the pane and when the mouse moves over
+        // the button, the tool tip will display.
+        private void CustomShapePane_MouseMove(object sender, MouseEventArgs e)
```

Comment:
The button can be embedded inside a grid, with hitTest enabled and has the tooltip for the button. This is easier done with WPF (in fact I see that tooltip service can remove the need for a grid), not sure if its difficult to do currently.

Reply:
Might want to ignore this if its too difficult, current implementation is fine too,

Reply:
I think @YuPeiHenry's suggestion is to have a different implementation for this tooltip of the disabled button, so that you won't have to constantly track the movement of the mouse (which seems inefficient in a way).

Reply:
I also think it could be easier to do it in WPF so... might want to change this implementation when moving over to WPF.

Reply:
@leeyh20 Yeah the `SyncLab` implementation is much simpler!